lib/std/elf.zig

```diff
@@ -455,6 +455,54 @@ pub const ET = enum(u16) {
     pub const HIPROC = 0xffff;
 };
 
+/// All integers are native endian.
+pub const SectionHeader = struct {
```

Comment:
Opinionated code like this should not go into std.elf but somewhere else such as std.zig.elf perhaps. However, why is this even needed? It's redundant with Elf64_Shdr.